lib/events/events.proto

```diff
@@ -61,6 +61,11 @@ message ServerMetadata {
     // session occurred on.
     map<string, string> ServerLabels = 5
         [ (gogoproto.nullable) = false, (gogoproto.jsontag) = "server_labels,omitempty" ];
+
+    // Kubernetes has information about a kubernetes cluster, if this was a
+    // kubernetes server request.
+    KubernetesClusterMetadata Kubernetes = 6
```

Comment:
I'm not sure TBH that "server metadata" is the best place to add Kubernetes cluster name... As I understand it (and I may be mistaken), it is supposed to contain information about particular cluster node you're teleporting onto, while K8s cluster name is something "global" to a cluster, outside of a single server scope.

Have you considered adding `KubernetesMetadata` as a separate field to those session events? That's what AAP did I think (and I'm doing the same for db).

Reply:
Makes sense, moved to a separate field.